Cargo.toml

```diff
@@ -89,6 +89,13 @@ zlib-ng = ["libc", "cmake"]
 # or be completely removed in the future
 zlib-ng-no-cmake-experimental-community-maintained = ["libc"]
 stock-zlib = []
+# Experimental: This feature only affects zlib-ng, is only relevant to riscv64
+# targets, and currently only has an effect when `cmake` is used. It builds
+# binaries portable to machines without RVV, even if the build machine is
+# riscv64 and detected to support RVV. This is useful for building a more
+# portable binary, or on systems where auto-detection is incorrect
+# (as in https://github.com/zlib-ng/zlib-ng/issues/1705).
```

Comment:
I think here one would also have to advertise the `RVV_OFF` environment variable.

Reply:
I've made the same change in both places, per https://github.com/rust-lang/libz-sys/pull/218#discussion_r1749197667.